File: data_structure_and_algorithms/leetcode/c/twoSum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target) {
    int *retList = (int *)malloc(sizeof(int) * 2);
    int i, j;
    
    for (i = 0; i < numsSize - 1; ++i) {
        for (j = i; j < numsSize - 1; ++j) {
            if (nums[i] + nums[j + 1] == target) {
                retList[0] = i;
                retList[1] = j + 1;
                return retList;
            }       
        }
    }
    return retList;
}

File: data_structure_and_algorithms/leetcode/python/twoSum.py
def twoSum(nums, target):
    retList = [] 
    for i in range(0, len(nums) - 1):
        for j in range(i, len(nums) - 1):
            if nums[i] + nums[j + 1] == target:
                retList.append(i)
                retList.append(j + 1)
                return retList
    return retList